rpc/tester/tester_test.go

```diff
@@ -491,3 +491,31 @@ func TestEth_GetLogs_Topics_AB(t *testing.T) {
 
 	require.Equal(t, 1, len(logs))
 }
+
+func TestEth_NewPendingTransactionFilter(t *testing.T) {
+	rpcRes, err := call(t, "eth_newPendingTransactionFilter", []string{})
+	require.NoError(t, err)
+
+	var code hexutil.Bytes
+	err = code.UnmarshalJSON(rpcRes.Result)
+	require.NoError(t, err)
+	require.NotNil(t, code)
+
+	for i := 0; i < 5; i++ {
+		deployTestContractWithFunction(t)
+	}
+
+	time.Sleep(10 * time.Second)
```

Comment:
I think this might be too long, when I tried running this I got:
```
--- FAIL: TestEth_NewPendingTransactionFilter (10.12s)
    tester_test.go:73: Post http://localhost:8545: http: server closed idle connection
FAIL
FAIL	github.com/cosmos/ethermint/rpc/tester	10.153s
FAIL
```

but when I reduced it to 1s I got:
```
--- FAIL: TestEth_NewPendingTransactionFilter (1.10s)
    tester_test.go:519: 
        	Error Trace:	tester_test.go:519
        	Error:      	Should be true
        	Test:       	TestEth_NewPendingTransactionFilter
        	Messages:   	could not get any txs%!(EXTRA string=changesRes.Result, string=["0xa3e5482e05c0b5e03e99788e399a08076f3cc55f70f62bdf77abe73b123f347f"])
FAIL
FAIL	github.com/cosmos/ethermint/rpc/tester	1.136s
FAIL
```

Reply:
yes, well...what can I do. The whole testing framework needs a overhaul that will be done on #253